commit d382c6e09d9811003c123cbd5411433f4024f1ff
Author: Trent Mick <trentm@gmail.com>
Date:   2019-07-16T15:55:54-07:00 (3 months ago)
    
    TRITON-1818 explicitly specify PEM key format for keys generated by ssh-keygen

diff --git a/docs/index.md b/docs/index.md
index e9ed3b1a5..2fa389d65 100644
--- a/docs/index.md
+++ b/docs/index.md
@@ -133,7 +133,7 @@ to many instances.
 If you haven't already generated an SSH key (required to use both SSH and HTTP
 Signing), run the following command:
 
-    $ ssh-keygen -b 2048 -t rsa
+    $ ssh-keygen -b 2048 -t rsa -m PEM
 
 This will prompt you with a place to save the key.  You should probably just
 accept the defaults, as many programs (SSH and CloudAPI CLIs) will first look
diff --git a/test/volumes-automount.test.js b/test/volumes-automount.test.js
index e47abea2e..d7d956230 100644
--- a/test/volumes-automount.test.js
+++ b/test/volumes-automount.test.js
@@ -215,6 +215,7 @@ if (CONFIG.experimental_cloudapi_nfs_shared_volumes !== true) {
         child_process.exec([
             'ssh-keygen',
             '-t rsa',
+            '-m PEM',
             '-N ""',
             '-f',
             KEY_FILENAME
